### PR TITLE
Add/animations

### LINKED ISSUE #164: Feat: Adding Animation
### Describe the feature

- Your landing page already looks fantastic, but incorporating animations could make it even more captivating. With my experience in this area, I'm eager to contribute to this enhancement.
- If you could assign me this task under GSSOC24, I would be thrilled to work on it.

### Add ScreenShots

EXAMPLE

https://github.com/Nactore-Org/Nacto-Care/assets/115486960/cbf75b8f-a7f3-4877-9b01-20442cc81c68

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by harshmishra19**
## Related Issue
This PR fixes #164 

## Description
I have added animations

## Type of PR

- [ ] Bug fix
- [ x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

https://github.com/Nactore-Org/Nacto-Care/assets/115486960/10c0c510-d465-4097-bba9-a0e31151bfcb

## Checklist:
- [x ] I have performed a self-review of my code
- [x ] I have read and followed the Contribution Guidelines.
- [x ] I have tested the changes thoroughly before submitting this pull request.
- [x ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x ] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

File: vanilla-tilt.js
Comment (Harshil-Jani) [inline vanilla-tilt.js:1]: Do we really need this file ? Can't we use npm package of vanilla tilt ?

Comment (harshmishra19) [reply vanilla-tilt.js:1]: Yes we can

**Comment by harshmishra19** [comment]
@Harshil-Jani If you want I'll remove that file

**Comment by Harshil-Jani** [comment]
@harshmishra19 Please remove it.

**Comment by harshmishra19** [comment]
Any update @Harshil-Jani

**Comment by Harshil-Jani** [comment]
@harshmishra19 Just resolve the conflicts and then we are good to go.

**Comment by harshmishra19** [comment]
@Harshil-Jani done

**Pull request merged**